src/multibody/fcl.hpp

```diff
@@ -189,23 +152,27 @@ struct GeometryObject
   /// \brief Name of the geometry object
   std::string name;
 
+  /// \brief Index of the parent frame
+  FrameIndex parentFrame;
```

Comment:
> parentFrame should be of type BODY
I agree, however, I don't think it should be mandatory to have a parentFrame. I would rather have a condition like `parentFrame == -1 || getFrameType(parentFrame) == BODY`